components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -209,6 +209,12 @@ be lost if SCREAM_HACK_XML is not enabled.
     <!-- CLD fraction -->
     <cldFraction inherit="atm_proc_base"/>
 
+    <!-- ML correction -->
+    <mlcorrection inherit="atm_proc_base">
+      <ML_model_path type="array(string)">NONE</ML_model_path>
```

Comment:
Is `array(string)` intended? "model_path" makes me think this is just a string...

Reply:
Yes you're right, this should be just a string.